docs/options.md

```diff
@@ -361,9 +361,9 @@ Valid options:
 - `"auto"` - Maintain existing line endings
   (mixed values within one file are normalised by looking at what's used after the first line)
 
-| Default | CLI Override                      | API Override                     |
-| ------- | --------------------------------- | -------------------------------- |
-| `"lf"`  | `--end-of-line <lf|crlf|cr|auto>` | `endOfLine: "<lf|crlf|cr|auto>"` |
+| Default | CLI Override                                                | API Override                                               |
+| ------- | ----------------------------------------------------------- | ---------------------------------------------------------- |
+| `"lf"`  | <code>--end-of-line <lf&#124;crlf&#124;cr&#124;auto></code> | <code>endOfLine: "<lf&#124;crlf&#124;cr&#124;auto>"</code> |
```

Comment:
Is it a bug in `docusaurus`, if it can't render correctly?

Reply:
Probably.